### DISCUSSION TITLE
Suspected harmful user downloading infinitely with `-DT0001-` `-GT0003-` and `github.com/anacrolix/torrent`

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [c0re100/qBittorrent-Enhanced-Edition#532] [General]
I noticed that a certain ip is downloading various torrent from me over and over again, with a really high upload speed

![image](https://github.com/c0re100/qBittorrent-Enhanced-Edition/assets/15852717/0ddd696b-102e-46c7-a616-a12f8655adeb)

I found relative discussions in the internet on [cometbbs](https://www.cometbbs.com/t/91488) and [anacrolix/torrent](https://github.com/anacrolix/torrent/discussions/891) and [v2ex](https://global.v2ex.com/t/1014159)

It seems that it's the same set of ips behind these clients, with names `dt/torrent/v1.00` `dt/torrent/v1.01` `dt/torrent/v1.02` and `github.com/anacrolix/torrent (devel)`, suspectedly a bugged out netdisk offline downloader.

Is there anything we can do about this?

**Comment by lzhzh1** [comment]
更新了！
Changelog
1.Official Update (v4.6.4)
2.Auto ban dt/torrent and Taipei-torrent (from China)
Just a cat and mouse game :|
